css/home.css

```diff
@@ -90,11 +90,20 @@
   bottom: 20px;
   right: 0;
   left: 0;
-  width: 33px;
+  width: 34px;
+  height: 40px;
   position: absolute;
   margin: auto;
 }
 
+.homepage-hero-anchor {
```

Comment:
this class should be something like `homepage-hero__anchor` to follow bem conventions

Reply:
done